src/Arcus.Messaging.Health/Tcp/TcpHealthListener.cs

```diff
@@ -148,7 +148,7 @@ private async Task AcceptConnectionAsync(HealthReport report, CancellationToken
             try
             {
                 _logger.LogTrace("Accepting TCP client on port {Port}...", Port);
-#if NET6_0
+#if !NETSTANDARD2_1 && !NETCOREAPP
```

Comment:
Made this currently still on .NET Core, as there is not yet a preview .NET 8 on this repo.

Reply:
Yes, but haven't removed the .NET Core yet. Wanted to keep the .NET 8 changes a bit outside the scope of this PR, and pure focus on the removal of deprecated code.